__tests__/api.test.js

```diff
@@ -72,6 +70,30 @@ describe('GET - /api/articles/:article_id', () => {
   });
 })
 
+describe("PATCH", () => {
+  test("responds with status 200 and spcific article is updated", () => {
+    return request(app)
+      .patch("/api/articles/2")
+      .send({ inc_votes: 50 })
+      .expect(200)
+      .then((res) => {
+        expect(res.body.article).toEqual(
+          expect.objectContaining({
+            article_id: 2,
+            title: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number)

```

Comment:
Will need here to check that the vote count has been increased by the amount specified in the pull request. For example, if it was 50 before, you'd expect it to be 100 after the patch 🙂